From ab6df0232350fca283bf8790cdcb1a6944c61ef9 Mon Sep 17 00:00:00 2001
From: Jean-Francois Roy <jf@devklog.net>
Date: Sun, 14 Jul 2024 09:21:16 -0700
Subject: [PATCH 2/4] update linux to linux 6.10.3

---
 Pkgfile | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/Pkgfile b/Pkgfile
index cda16a7..98682f6 100644
--- a/Pkgfile
+++ b/Pkgfile
@@ -68,9 +68,9 @@ vars:
   ipxe_sha512: f83d49da507db7810affb5a39abae732e7cd856b55c167522d4354a9ef5844e2dfef0e57d2304d139144d0b4773bf12ac5aeee8b9d2ab219d6b5da6757ba745b
 
   # renovate: datasource=git-tags extractVersion=^v(?<version>.*)$ depName=git://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git
-  linux_version: 6.6.43
-  linux_sha256: 0ad83b1a1a780a1aad948d55aa55ee63c50c626f2d46910b9d2180028d100a5e
-  linux_sha512: 013027b921184cf7fee360e930472689c0ad202e06b4ddcc2b3207f52ea6244890d9ed267ffe5590335501318dd8386c873f038de875a70d637a58f1a405a55a
+  linux_version: 6.10.3
+  linux_sha256: fa5f22fd67dd05812d39dca579320c493048e26c4a556048a12385e7ae6fc698
+  linux_sha512: dd6061c5871d7b313e1b0eec791444b911f9d9d5e28b1d7a81a265708688808c157adfba92459511e9b786d09e3f96f5bc3c03980360061344ad8e225e165f6a
 
   # renovate: datasource=git-tags extractVersion=^v(?<version>.*)$ depName=git://git.kernel.org/pub/scm/utils/kernel/kmod/kmod.git
   kmod_version: 32
-- 
2.45.2

